requirements-dev.txt

```diff
@@ -54,6 +54,7 @@ oauthlib==2.1.0
 openpyxl==3.1.2; python_version >= '3.6'
 packaging==23.2; python_version >= '3.7'
 pillow==10.0.1; python_version >= '3.8'
+prophet==1.1.1; python_version >= '3.7'
```

Comment:
This should be added to the `Pipfile` and locked — which I could not get to happen so best of luck to you!

Reply:
Done.